i18n/de/docusaurus-plugin-content-docs/current/help-center/your-account/account-settings.mdx

```diff
@@ -488,6 +488,37 @@ Abhängig davon in welchem Status sich das Chat-Backup befindet, haben Sie zwei
 
 In beiden Fällen, stellen Sie sicher, dass Sie die erstellte Passphrase sicher aufheben.
 
+## Chat-Einstellungen
+
+### Verhalten der Eingabetaste ändern
+
+In diesem Abschnitt finden Sie eine Anleitung, wie Sie das Verhalten der Eingabetaste verändern können.
+
+:::info
+Diese Einstellung lässt sich für jedes Gerät einzeln vornehmen und wird auf die Standardeinstellung zurückgesetzt, wenn du dich über die Profileinstellungen abmeldest oder das Gerät unter „Verbundene Geräte“ in den Einstellungen entfernst. Nach der Anmeldung musst du die Einstellung dann erneut vornehmen.
+:::
+
+<Tabs>
+  <TabItem value="web" label="Web" default>
+  <ol>
+    <li>Um die Einstellungen zu öffnen, klicken Sie auf Ihr Profilbild, oder Ihre Initialen.</li>
```

Comment:
Muss da ein Komma zwischen Profilbild und Initialen?

Reply:
The second comma is wrong, please remove it

Reply:
Removed